### PR TITLE
Backport 2.5.2 changelog

### LINKED ISSUE #6749: Release Securedrop 2.5.2
This is a tracking issue for the release of SecureDrop 2.5.2

Tentatively scheduled as follows:

**Pre-release announcement:** 2023-02-16
**Release date:** 2022-02-23

**Release manager:** @zenmonkeykstop  
**Deputy release manager:** @cfm  
**Communications manager:** @nathandyer

_SecureDrop maintainers and testers:_ As you QA 2.5.2, please report back your testing results as comments on this ticket. File GitHub issues for any problems found, tag them "QA: Release".

Test debian packages will be posted on https://apt-test.freedom.press signed with [the test key](https://gist.githubusercontent.com/conorsch/ec4008b111bc3142fca522693f3cce7e/raw/2968621e8ad92db4505a31fcc5776422d7d26729/apt-test%2520apt%2520pubkey).

# [QA Matrix for 2.5.2](https://docs.google.com/spreadsheets/d/18qkpFYs9aHR_q1RDc9KVH96cWmYz_oLmtGh-HK1AkLg/edit#gid=1940373141)
# [Test Plan for 2.5.2](https://github.com/freedomofpress/securedrop/wiki/2.5.2-Test-Plan)

# Prepare release candidate (2.5.2~rc1)
- [ ] Link to latest version of Tails, including release candidates, to test against during QA
- [x] Prepare 2.5.2~rc1 release changelog
- [x] Branch off release/2.5.2 from release/2.5.1
- [x] Prepare 2.5.2~rc1
- [x] Build debs, preserving build log, and put up `2.5.2~rc1` on test apt server
- [x] Commit build log.

After each test, please update the QA matrix and post details for Basic Server Testing, Application Acceptance Testing and release-specific testing below in comments to this ticket.

# Final release
- [ ] Ensure builder in release branch is updated and/or update builder image
- [ ] Push signed tag 
- [ ] Pre-Flight: Test updater logic in Tails (apt-qa tracks the `release` branch in the LFS repo)
- [ ] Build final Debian packages(and preserve build log)
- [ ] Commit package build log to https://github.com/freedomofpress/build-logs
- [ ] Pre-Flight: Test that install and upgrade from 2.5.1 to 2.5.2 works w/ prod repo debs (apt-qa.freedom.press polls the `release` branch in the LFS repo for the debs)
- [ ] Flip apt QA server to prod status (merge to `main` in the LFS repo)
- [ ] Merge Docs branch changes to ``main`` and verify new docs build in securedrop-docs repo
- [ ] Prepare release messaging

# Post release
- [ ] Create GitHub release object 
- [ ] Once release object is created, update versions in `securedrop-docs` and Wagtail
- [ ] Verify new docs show up on https://docs.securedrop.org
- [ ] Publish announcements
- [ ] Merge changelog back to `develop`
- [ ] Update roadmap wiki page: https://github.com/freedomofpress/securedrop/wiki/Development-Roadmap

### PR DESCRIPTION
**Pull request by zenmonkeykstop**
## Status

Ready for review 

## Description of Changes

Fixes #6749 .

- Backports 2.5.2 changelog (including RCs for historical edification)
- Updates version...updater with new location for setup.py

## Testing

- [x] CI is passing
- [x] changelog looks correct
- [x] `./update_version.sh <dummy version>` works as expected.

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -2,6 +2,23 @@
...
+
+## 2.5.2
+
+### Web applications
+- Updated dependencies:
+  - werkzeug from 2.0.2 to 2.2.3, markupsafe from 2.0.1 to 2.1.2, Flask-SQLAlchemy from 2.4.0
+ to 2.5.1 (#6752)
+  - cryptography from 3.4.7 to 39.0.1 (#6746, #6751)
+  - certifi from 2017.7.27.1 to 2022-12.07 (#6708)
+  - flask from 2.0.2 to 2.0.3 (#6624,#6634)
```
Comment (cfm) [inline changelog.md]: I didn't want to hold up #6758 for this tiniest of nits, but here I will.  @zenmonkeykstop, let me know by the end of the day today if you have any objections to breaking the strict backport-ness of this pull request.  :-)

```suggestion
  - flask from 2.0.2 to 2.0.3 (#6624, #6634)
```

**Review by cfm — DISMISSED** [review]
Looks good, @zenmonkeykstop, with one nit inline for you to feel free to block.  :-)

- [x] CI is passing
- [x] changelog looks correct
- [x] `./update_version.sh <dummy version>` works as expected.

```sh-session
user@sd-dev:~/securedrop$ securedrop/bin/dev-shell ../update_version.sh 7.0.0
[backport-2.5.2-changelog 286de664b] 7.0.0
 7 files changed, 15 insertions(+), 5 deletions(-)
[ok] Version update complete and committed.
A tag has been generated: 7.0.0.tag
If you wish to release this version, please continue the airgapped signing process with the tag file.
user@sd-dev:~/securedrop$ git show
commit 286de664b4caf9f9ca0fb91196d859b29cac9cac (HEAD -> backport-2.5.2-changelog, tag: 7.0.0)
Author: Cory Francis Myers <cory@freedom.press>
Date:   Wed Mar 1 19:45:21 2023 +0000

    7.0.0

diff --git a/changelog.md b/changelog.md
index 03ea49de7..d2fd90081 100644
--- a/changelog.md
+++ b/changelog.md
@@ -1,5 +1,9 @@
 # Changelog
 
+## 7.0.0
+
+
+
 ## 2.6.0~rc1
 
 
diff --git a/install_files/ansible-base/group_vars/all/securedrop b/install_files/ansible-base/group_vars/all/securedrop
index 3fa40776e..ee7dc3cc6 100644
--- a/install_files/ansible-base/group_vars/all/securedrop
+++ b/install_files/ansible-base/group_vars/all/securedrop
@@ -2,7 +2,7 @@
 # Variables that apply to both the app and monitor server go in this file
 # If the monitor or app server need different values define the variable in
 # hosts_vars/app.yml or host_vars/mon.yml
-securedrop_version: "2.6.0~rc1"
+securedrop_version: "7.0.0"
 securedrop_app_code_sdist_name: "securedrop-app-code-{{ securedrop_version | replace('~', '-') }}.tar.gz"
 
 grsecurity: true
diff --git a/molecule/builder-focal/tests/vars.yml b/molecule/builder-focal/tests/vars.yml
index 81eda8b01..1b0230a0f 100644
--- a/molecule/builder-focal/tests/vars.yml
+++ b/molecule/builder-focal/tests/vars.yml
@@ -1,5 +1,5 @@
 ---
-securedrop_version: "2.6.0~rc1"
+securedrop_version: "7.0.0"
 ossec_version: "3.6.0"
 keyring_version: "0.1.6"
 
diff --git a/molecule/shared/stable.ver b/molecule/shared/stable.ver
index f225a78ad..66ce77b7e 100644
--- a/molecule/shared/stable.ver
+++ b/molecule/shared/stable.ver
@@ -1 +1 @@
-2.5.2
+7.0.0
diff --git a/securedrop/debian/changelog b/securedrop/debian/changelog
index 5cab0bb37..9855f0a37 100644
--- a/securedrop/debian/changelog
+++ b/securedrop/debian/changelog
@@ -1,3 +1,9 @@
+securedrop (7.0.0+focal) focal; urgency=medium
+
+  * 
+
+ -- SecureDrop Team <securedrop@freedom.press>  Wed, 01 Mar 2023 19:45:19 +0000
+
 securedrop (2.6.0~rc1+focal) focal; urgency=medium
 
   * 
diff --git a/securedrop/setup.py b/securedrop/setup.py
index 6c5d051f9..84843422b 100644
--- a/securedrop/setup.py
+++ b/securedrop/setup.py
@@ -4,7 +4,7 @@ long_description = "The SecureDrop whistleblower platform."
 
 setuptools.setup(
     name="securedrop-app-code",
-    version="2.6.0~rc1",
+    version="7.0.0",
     author="Freedom of the Press Foundation",
     author_email="securedrop@freedom.press",
     description="SecureDrop Server",
diff --git a/securedrop/version.py b/securedrop/version.py
index aed085141..b378fc6cc 100644
--- a/securedrop/version.py
+++ b/securedrop/version.py
@@ -1 +1 @@
-__version__ = "2.6.0~rc1"
+__version__ = "7.0.0"
```

File: changelog.md (hunk 1 above)
Comment (zenmonkeykstop) [reply changelog.md]: No issues, go for it.

**Review by cfm — APPROVED** [review]
Thanks, @zenmonkeykstop!

**Pull request merged**